### ISSUE TITLE
Triage process for new issues

### ISSUE THREAD

**Issue by jweill-aws** [jupyterlab/jupyterlab#11660]
### Problem

As I write this there are over 1,900 issues in this project, with new ones coming in regularly. A **triage** process will provide a mechanism for us to give attention to new issues by accepting those that meet our definition of ready and by asking for additional information or closing issues where appropriate.

### Proposed Solution

1. Add a new automation in GitHub that applies the `status: Needs Triage` label to all new issues filed.
2. Update the documentation to refer to the triage process.
3. (optional) Metrics to track the number of items that have `status: Needs Triage` or `status: Needs Info` labels, over time.

### Additional context

We discussed this at the December 8, 2021 JupyterLab meeting. Notes: https://github.com/jupyterlab/team-compass/issues/128#issuecomment-989050676

This is a follow-up to issue https://github.com/jupyterlab/team-compass/issues/90 and its pull request https://github.com/jupyterlab/team-compass/pull/134.

**Comment by krassowski** [comment]
Three questions:
- this is a duplicate of #5726; should we consolidate the discussion by closing one or the other?
- did the pull request #11661 resolve this issue?
- https://github.com/jupyterlab/team-compass/issues/135 mentions a triage meeting; is it documented beyond the main meeting notes and the calendar? Should it be mentioned more prominently? Maybe a pinned issue, a note in the team-compass readme?

**Comment by jweill-aws** [comment]
@krassowski Thanks for following up about this!

1. I can close the older issue in favor of this one. Thanks for finding it!
2. Yes. If we need to make other changes, we can do so in a separate issue.
3. We've already changed the triage meeting's time slot once and we might do so in the future. I'll set a reminder to add this to the team-compass readme once we've iterated a little bit on the process. In the meantime, we'll continue to mention it at the JupyterLab weekly meetings on Wednesdays at 09:00 PST.
